src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.main-h1 {
```

Comment:
better to use names like 'main-heading' than using names of the tags in class names